core/src/main/java/com/linecorp/armeria/internal/client/DefaultClientRequestContext.java

```diff
@@ -328,8 +328,9 @@ public CompletableFuture<Boolean> init(EndpointGroup endpointGroup) {
                 return initEndpointGroup(endpointGroup);
             }
         } catch (Throwable t) {
-            acquireEventLoop(endpointGroup);
-            failEarly(t);
```

Comment:
I wonder why we `acquireEventLoop` when exception occur during the init phase 🤔

Reply:
What do you think about not raising an exception in `acquireEventLoop`? (Apologize for my previous comment. 😓 )
I think we need to let the caller acquire an event loop to use it for handling exceptions, even in closing situations.

Reply:
Aha, so we need to call `acquireEventLoop` to handle anything that's related to request? 

I've removed the code which raises exceptions in `acquireEventLoop`